lang/python/python3/Makefile

```diff
@@ -282,6 +285,9 @@ define Py3Package/python3-base/install
 	$(LN) python$(PYTHON3_VERSION) $(1)/usr/bin/python3
 	$(LN) python$(PYTHON3_VERSION) $(1)/usr/bin/python
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpython$(PYTHON3_VERSION).so* $(1)/usr/lib/
+  # This depends on being called before filespec is processed
```

Comment:
nitpick: minor indentation inconsistency on comment;

Reply:
A comment with spaces is interpreted as a Makefile comment, whereas a comment with tab is interpreted as a command to be passed to the shell (which interprets it as a comment). Since this comment is relevant for Makefile maintenance and not for the actual build, I would prefer to not echo it during install. (This isn’t a strong preference though, I can change it if you really prefer tabs.)

Reply:
right; my bad
nvm